docs/model-binding.md

```diff
@@ -224,7 +224,7 @@ var command = new RootCommand
 command.Handler = CommandHandler.Create(
     (ParseResult parseResult, IConsole console) =>
     {
-        console.Out.WriteLine($"{parseResult}");
+        Console.Out.WriteLine($"{parseResult}");
```

Comment:
While this change will work, the existing sample also works. `console` refers to the parameter of type `IConsole` being passed into the handler.

Reply:
Oh that's right. I'll close the pull request. Sorry